syditest/test_api.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import requests
+import re
+import sys
+import atexit
+
+# These are standard python unit tests, but are generally intended
+# to be run with trial. Trial doesn't capture logging nicely if you
+# use python 'logging': it only works if you use Twisted's own.
+from twisted.python import log
+
+from .mailsink import MailSink
+
+try:
+    from syditest_subject.launcher import SyditestLauncher
+except ImportError:
+    print("ERROR: Couldn't import launcher")
+    print(
+        "syditest needs an identity server to test: make sure, "
+        "'syditest_subject.launcher.SyditestLauncher' is in "
+        "sys.path"
+    )
+    
+    raise
+
+launcher = None
+baseUrl = None
+
+def getOrLaunchIS():
```

Comment:
please use underscores for methods and variables. CamelCase is good for classes.